components/AmountInput.js

```
@@ -206,139 +301,217 @@
+                  {...this.props}
+                  caretHidden
+                  testID="BitcoinAmountInput"
+                  keyboardType="numeric"
+                  adjustsFontSizeToFit
+                  onChangeText={this.handleChangeText}
+                  onBlur={this.handleBlur}
+                  onFocus={() => {
+                    if (this.props.onFocus) this.props.onFocus();
+                  }}
+                  maxLength={this.maxLength()}
+                  ref={textInput => (this.textInput = textInput)}
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)